src/content/dataSubmissions/DataSubmission.tsx

```diff
@@ -151,6 +156,48 @@ const StyledRejectedStatus = styled("div")(() => ({
   fontWeight: 600
 }));
 
+const StyledCopyWrapper = styled(Stack)(() => ({
+  height: "42px",
+  width: "fit-content",
+  minWidth: "342px",
+  padding: "11px 20px",
+  borderRadius: "8px 8px 0px 0px",
+  borderTop: "1.25px solid #6DADDB",
+  borderRight: "1.25px solid #6DADDB",
+  borderLeft: "1.25px solid #6DADDB",
+  background: "#EAF5F8",
+}));
+
+const StyledCopyLabel = styled(Typography)(() => ({
+  color: "#125868",
+  fontFamily: "'Nunito', 'Rubik', sans-serif",
+  fontSize: "12px",
+  fontStyle: "normal",
+  fontWeight: 800,
+  lineHeight: "19.6px",
+  letterSpacing: "0.24px",
+  textTransform: "uppercase",
+}));
+
+const StyledCopyValue = styled(Typography)(() => ({
+  color: "#125868",
+  fontFamily: "'Nunito', 'Rubik', sans-serif",
+  fontSize: "16px",
+  fontStyle: "normal",
+  fontWeight: 400,
+  lineHeight: "19.6px",
+  letterSpacing: "0.32px",
+  textTransform: "uppercase",
```

Comment:
I think transforming the UUID to uppercase may be an issue if the user tries to manually copy it. The backend does seem to treat them as case-sensitive.